docs/api-routes/dynamic-api-routes.md

```diff
@@ -13,88 +13,88 @@ description: You can add the dynamic routes used for pages to API Routes too. Le
 
 API routes support [dynamic routes](/docs/routing/dynamic-routes.md), and follow the same file naming rules used for `pages`.
 
-For example, the API route `pages/api/post/[pid].js` has the following code:
+For example, the API route `pages/api/note/[pid].js` has the following code:
```

Comment:
This should be `[nid]` instead of `[pid]` since you used `{ nid }` below

Reply:
Thanks for pointing these out, missed them. I have made the appropriate changes